alphacsc/datasets/mne_data.py

```diff
@@ -81,7 +83,7 @@ def load_data(dataset="somato", n_splits=10, sfreq=None, epoch=None,
 
     baseline = (None, 0)
     events = mne.find_events(raw, stim_channel='STI 014')
-    events = mne.pick_events(events, include=event_id)
+    events = mne.pick_events(events, include=list(event_id.values()))
 
     # compute the covariance matrix for somato
     if dataset == "somato":
```

Comment:
Avoid unrelated changes ;)
It is okay to let them be but try to avoid putting them in PR.